snappy_wrappers/wrappers/varfish_annotator/annotate_sv/wrapper.py

```diff
@@ -59,13 +66,16 @@
 
 samples=$(cut -f 2 {snakemake.input.ped} | tr '\n' ',' | sed -e 's/,$//g')
 
+# Fix the Manta inversions
+python3 {fix_manta_invs} {snakemake.config[static_data_config][reference][path]} {snakemake.input.vcf} $TMPDIR/tmp4.vcf
+bcftools sort -o $TMPDIR/tmp3.vcf $TMPDIR/tmp4.vcf
```

Comment:
Is it possible to use more meaningful names? `tmp*.vcf` is confusing.

Reply:
doen